task/clair-scan/0.1/clair-scan.yaml

```diff
@@ -58,8 +60,9 @@ spec:
         . /utils.sh
 
         if [[ ! -f /tekton/home/clair-vulnerabilities.json ]] || [[ "$(jq '.[] | has("failures")' /tekton/home/clair-vulnerabilities.json)" == "false" ]]; then
-          HACBS_TEST_OUTPUT=$(make_result_json -r "ERROR" -t "/tekton/home/clair-vulnerabilities.json is not generated correctly, please check again")
-          echo "/tekton/home/clair-vulnerabilities.json is not generated correctly, please check again"
+          note="Task \`$(context.task.name)\` failed: \`/tekton/home/clair-vulnerabilities.json\` did not generate correctly. For details, check the Tekton task log."
+          HACBS_TEST_OUTPUT=$(make_result_json -r "ERROR" -t "$note")
+          echo "/tekton/home/clair-vulnerabilities.json is not generated correctly, check the the previous log in this task."
```

Comment:
@jseseCCS This is a log in task log, is it proper if we still ask users to `check the Tekton task log.` , I feel users might think 'I am already reading task log, why you ask me to check log again?'

Reply:
ah ha! i didn't understand that. thank you!! so is there anywhere that a user can find out more, @hongweiliu17?

Reply:
I change it to `echo "/tekton/home/clair-vulnerabilities.json did not generate correctly. For details, check output of conftest command in Tekton task log."` since there might be some log in output of conftest command.